src/composables/useMap.js

```diff
@@ -80,7 +80,7 @@ export const mapReady = apply(map, './map/style.json').then(() => {
     getSource(map, layer.source).tileOptions.transition = undefined;
   });
   getLayer(map, 'neigungsklassen').setSource(null);
-  getSource(map, 'agrargis').overlaps_ = false; // eslint-disable-line
+  // getSource(map, 'agrargis').overlaps_ = false; // eslint-disable-line
```

Comment:
Candidate for removal - looks like modern browsers perform worse with the `overlaps: false` optimisation.